sklearn/base.py

```diff
@@ -320,16 +320,28 @@ def is_non_default(param_name, param_value):
 
         # reorder the parameters from `self.get_params` using the `__init__`
```

Comment:
I think that we need to update this comment stating that we are going to put the non-default first.